Frends.AmazonSQS.Receive/Frends.AmazonSQS.Receive.Tests/.env.example

```
@@ -0,0 +1 @@
+FRENDS_TEST_APP_ID="14bkhhd189011"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)